app/models/config.rb

```diff
@@ -41,6 +41,14 @@ def options
     config_value[:options]
   end
 
+  def fax_service
+    config_value[:service]
+  end
+
+  def fax_number
```

Comment:
You know, I think you're absolutely right on how the fax number here makes things much more complex. Let's ditch it and just go with fax service, and ask them to make that a url like `http://whatever.com`. Hopefully that cuts down on a little bit of the complexity/ugmo-ness. That way we can make it handle more like `resources_url`. thoughts?